resources/views/pages-legacy/viewtopic.blade.php

```diff
@@ -195,7 +195,7 @@
     <?php
     $user = auth()->user();
     ?>
-    @if ($thisTopicID != 0 && $user?->hasVerifiedEmail())
+    @if ($thisTopicID != 0 && $user?->hasVerifiedEmail() && !$user?->isMuted)
         <x-section>
             <div class="flex w-full bg-embed p-2 rounded-lg">
                 @guest
```

Comment:
Unrelated to this PR, but how can this `@guest` code be reached? A null user would probably not have a verified email, and definitely wouldn't be muted.

Reply:
Good catch, it's unreachable. I went ahead and fixed it in latest by moving it outside the `@if` directive.

Reply:
I see the new code at line 200, but the old (unreachable) code is still present here.

Reply:
Whoops, removed in latest.